entrypoint/fb.js

```diff
@@ -181,10 +181,13 @@ const handleMessage = async (event, context, chat, msgEvent) => {
         console.log(`  Parameters: ${JSON.stringify(result.parameters)}`);
         console.log(`  Action: ${result.action}`);
         if (result.action in handler.actions) {
+            chat.dialogflowParams = result.parameters.fields;
```

Comment:
these parameters are already passed as the payload to the handler. is there a reason you want them to also be set on the chat?

Reply:
used payload - also added parameter dialogparams in tg